### PR TITLE
ENH: test different cache TTLs for performance gains

### PR DESCRIPTION
**Pull request by eagerterrier**
let's experiment with longer TTLs for performance gains (TTFB etc)

- extend arabic's default TTL from 30s to 45s
- extend the TTL of topic pages to be double the default TTL (e.g. 90s for arabic and 60s for any other service)

### CODE REVIEW

File: src/server/index.jsx [hunk 1]
```diff
@@ -50,6 +53,10 @@ class LoggerStream {
   }
 }
 
+const getDefaultMaxAge = req => {
+	return req.originalUrl.indexOf('arabic/') !== -1 EXPERIMENTAL_CACHE_TTL : NORMAL_CACHE_TTL;
```
Comment (amoore108) [inline src/server/index.jsx]: Just a missing `?` in the ternary.

Comment (eagerterrier) [reply src/server/index.jsx]: wow! Thanks, Aaron. Fixed

File: src/server/index.jsx [hunk 2]
```diff
@@ -126,9 +133,13 @@ if (process.env.SIMORGH_APP_ENV === 'local') {
...
 
 const injectDefaultCacheHeader = (req, res, next) => {
+  const defaultMaxAge = getDefaultMaxAge(req);
+  const maxAge = req.originalUrl.indexOf('/topics/') !== -1 ? defaultMaxAge * 2 : defaultMaxAge;
   res.set(
-    'cache-control',
-    `public, stale-if-error=90, stale-while-revalidate=30, max-age=30`,
+    'Cache-Control',
+    `public, stale-if-error=${
+      maxAge * 4
```
Comment (amoore108) [inline src/server/index.jsx:146]: Is the `*4` needed here?

Comment (eagerterrier) [reply src/server/index.jsx:146]: I thought we'd multiply whichever maxAge by 4 in order to have a longer cache when the server responds with a non-200 response

e.g. for 45s we'd serve stale for 180s

Does that make sense or have I gone off the rails here?

Comment (amoore108) [reply src/server/index.jsx:146]: Ah yea makes sense. I read that as the value for `max-age`. Linter taking new lines confused me!

**Review by amoore108 — APPROVED** [review]

File: src/server/index.jsx (hunk 2 above)
Comment (amoore108) [reply src/server/index.jsx:146]: Just some lint fixes needed

Comment (eagerterrier) [reply src/server/index.jsx:146]: grand. Thanks

I've run the idea passed Neil and he's happy with it, so I'll tidy this up and then we can launch

Comment (eagerterrier) [reply src/server/index.jsx:146]: Launching tomorrow so we can see any effects from today's deployments

**Pull request merged**